packages/sil-governance/src/routes/create-proposal/index.tsx

```diff
@@ -1,19 +1,37 @@
 import { Block, Hairline } from "medulas-react-components";
 import React from "react";
+import { useSelector } from "react-redux";
 
+import { history } from "..";
+import { store } from "../..";
 import AsideFilter from "../../components/AsideFilter";
+import ConfirmTransaction from "../../components/ConfirmTransaction";
 import Header from "../../components/Header";
+import { RootState } from "../../store/reducers";
+import { setTransactionsStateAction } from "../../store/transactions";
+import { DASHBOARD_ROUTE } from "../paths";
 import ProposalForm from "./components/ProposalForm";
 
+const onReturnToDashboard = (): void => {
+  store.dispatch(setTransactionsStateAction());
+  history.push(DASHBOARD_ROUTE);
+};
```

Comment:
You can move this function into the component and then use the default way to get the dispatch: `const dispatch = ReactRedux.useDispatch();`